src/js/browser-plugin.js

```diff
@@ -1194,6 +1211,9 @@ function openLabelPopup(tweet) {
     confirmButtonText: buttonText,
     reverseButtons: true,
     focusCancel: true,
+    onClose: () => {
+      log2Server('explainability', node.tweetUrl, `Tweet id: ${node.tweetid}\nTweet label: ${node.coInformLabel}`, 'click on explainability popup close button');
```

Comment:
I tested it and both node.tweetUrl and node.tweetid are undefined.
Please test and debug the different logs so each one is called when it should and the parameters are as expected.
I recommend to just set a breakpoint on line 1550 (inside function "log2Server"), and do the different actions that would generate the logs, and check what log is going to be generated at such code point.